Glossario.json

```diff
@@ -23,6 +23,11 @@
     "description": "Entità che interagisce con il sistema svolgendo determinate azioni.",
     "acronyms": [],
     "synonyms": []
+  },
+	"Azure": {
+    "description": "Piattaforma di servizi cloud fornita da Microsoft che permette di distribuire applicazioni e servizi su cloud",
```

Comment:
- Per evitare ripetizioni riformulerei in "Piattaforma fornita da Microsoft che permette di distribuire applicazioni e servizi su cloud.";
- Manca il punto a fine definizione.